cardano-cli/src/Cardano/CLI/Shelley/Commands.hs

```diff
@@ -185,7 +186,7 @@ data TransactionCmd
       [ScriptFile]
       -- ^ Auxiliary scripts
       [MetadataFile]
-      (Maybe ProtocolParamsFile)
+      (Maybe ProtocolParamsSource)
```

Comment:
build-raw is also an offline cli command

Reply:
You can't use `ProtocolParamsSource` here because it's an offline command i.e you do not have to run a node to use it.